javascript-javascript1-week3/remove.js

```diff
@@ -0,0 +1,24 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+// Write some code here
+//const nameToRemove = names.splice(1, 1);
+// Code done
+
+//console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+//console.log(nameToRemove); // 
+
+const nameToRemove = "Ahmad";
+
+const indexToRemove = names.indexOf(nameToRemove);
+names.splice(indexToRemove, 1);
```

Comment:
Good dynamic implementation. One thing you could do to improve the code is checking if `indexToRemove` is different than -1. If there is no match from `indexOf` then it will return exactly -1 and this will basically remove the last item from the array, which would be unintentional.
You can test it by changing `nameToRemove` to a name that doesn't exist in the array and you will see how `Tala` will disappear.